tello Test/tello_web_server.py

```
@@ -651,15 +740,650 @@ def _log_operation(self, operation: str, details: Dict[str, Any]):
+                    self.cap.set(cv2.CAP_PROP_BUFFERSIZE, 1)  # バッファサイズを最小に
+                    self.cap.set(cv2.CAP_PROP_FPS, 30)  # フレームレートを向上
+                    
+                    # タイムアウト設定を追加
+                    self.cap.set(cv2.CAP_PROP_OPEN_TIMEOUT_MSEC, 5000)  # 5秒タイムアウト
+                    self.cap.set(cv2.CAP_PROP_READ_TIMEOUT_MSEC, 3000)  # 3秒読み取りタイムアウト
+                    
+                    # フォーマット設定（より柔軟に）
+                    try:
+                        self.cap.set(cv2.CAP_PROP_FOURCC, cv2.VideoWriter_fourcc('H', '2', '6', '4'))
+                    except:
+                        pass  # フォーマット設定に失敗しても続行
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)